continuous_integration/home/coder/.local/bin/build-legate-conda

```diff
@@ -22,7 +22,11 @@ build_legate_conda_package() {
     conda_build_args+=("--build-id-pat=''");
     conda_build_args+=(--no-include-recipe);
     conda_build_args+=(--no-anaconda-upload);
-    conda_build_args+=(--variants "{gpu_enabled:true,python:${python_version}}");
+
+    GPU_ENABLED=true
+    [ "$USE_CUDA" = "OFF" ] && GPU_ENABLED=false
```

Comment:
I'm curious about what this line does, but otherwise LGTM.

Reply:
Essentially, this just sets the value of the GPU_ENABLED variable to false when USE_CUDA=OFF and true otherwise (i.e. when USE_CUDA=ON).